open_schools_platform/organization_management/circles/views.py

```diff
@@ -63,6 +64,11 @@ class GetCirclesApi(ApiAuthMixin, ListAPIView):
         tags=[SwaggerTags.ORGANIZATION_MANAGEMENT_CIRCLES],
     )
     def get(self, request, *args, **kwargs):
+        data = request.GET.dict()
+        if 'student' in data.keys():
+            get_student(filters={"id": data['student']}, user=request.user)
+            if 'organization' not in data.keys():
+                raise NotAcceptable("Organization is not defined")
```

Comment:
We don't use NotAcceptable exception now. Look for it in errors or common app

Reply:
should we check permission of organization?
I think yes

Reply:
But circles are publicly available, so I think everyone should be able to access list of organization's circles.
At least 'organization' query parameter already was here before my changes and it didn't require permissions.